rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
```

Comment:
>it is an information that is not fomally requested and most of the time not used.

I don't agree to this. Even when providing pagination it makes sense to provide this information also.

Imagine a frontend UI fetching items from db and displaying it in a table with page numbers. without total items count, how will you know how many page numbers to display?